src/main/java/wolox/training/services/UserService.java

```diff
@@ -32,6 +32,7 @@ public User create(User user) {
 
     public User update(User user, Long id) {
         userRepository.findById(id).orElseThrow(BookNotFoundException::new);
```

Comment:
Add validation to id vs user.getId()
`if (user.getId() != id) {
            throw new UserIdMismatchException();	           
        }`